documentation/debug.md

```diff
@@ -97,8 +97,8 @@ terminal]
 Now it is time to set the breakpoints. For example
 
 ```
-(gdb) b plat_tee_entry
-Breakpoint 1 at 0x7df104d6: file core/arch/arm/plat-vexpress/plat_tee_func.c, line 42.
+(gdb) b tee_entry
```

Comment:
Since plat_tee_entry isn't existing any longer, I think it's a good idea to update this (and the ones below).